### PR TITLE
Integrate discussions front end with server api

### PR DESCRIPTION
**Pull request by atstoyanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: graphql/queries/userAvatars.graphql [hunk 1]
```diff
@@ -1,7 +1,10 @@
 query userAvatars($ids: [UUID_NAMEID_EMAIL!]!) {
```
Comment (hero101) [inline graphql/queries/userAvatars.graphql:1]: I suggest using a different query for this one.

File: src/components/composite/common/Discussion/PostComment.tsx [hunk 2]
```diff
@@ -25,8 +25,9 @@ const PostComment: FC<PostCommentProps> = ({ onPost }) => {
   });
 
   const handleSubmit = async (values: formValues, _helpers: FormikHelpers<formValues>) => {
-    if (onPost) {
-      await onPost(values.post);
+    if (onPostComment) {
+      await onPostComment(values.post);
```
Comment (hero101) [inline src/components/composite/common/Discussion/PostComment.tsx:29]: This is not an async function in the props definition.

File: src/views/Discussions/NewDiscussionView.tsx [hunk 3]
```diff
@@ -59,14 +60,15 @@ const NewDiscussionView: FC<NewDiscussionViewProps> = ({ onPost }) => {
...
                 rows={10}
                 multiline
+                disabled={isSubmitting}
               />
             </Grid>
             <Grid item>
               <Button
                 variant="primary"
-                text={t('components.new-discussion.buttons.post')}
+                text={isSubmitting ? t('buttons.processing') : t('components.new-discussion.buttons.post')}
```
Comment (hero101) [inline src/views/Discussions/NewDiscussionView.tsx:69]: 'Processing' and 'Post' feel like a generic action to a button. Extract it to buttons instead.

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/views/Discussions/NewDiscussionView.tsx (hunk 3 above)
Comment (atstoyanov) [reply src/views/Discussions/NewDiscussionView.tsx:69]: Post is the action. the real name is 'Post comment'

**Review by hero101 — APPROVED** [review]

**Pull request merged**